backend-config/backend-config.go

```diff
@@ -340,6 +340,8 @@ func (bc *backendConfigImpl) StartWithIDs(ctx context.Context, workspaces string
 				bc.cache = &noCache{}
 			}
 		}
+	} else {
```

Comment:
else needs to be done only in case of canUseCache as false right?

Reply:
Otherwise, it will override the cache when it it initialized and canUseCache is set to true?